app/controllers/users/statistiques_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Users
+  class StatistiquesController < ApplicationController
+    def statistiques
+      @procedure = retrieve_procedure
+      return procedure_not_found if @procedure.blank? || @procedure.brouillon?
+
+      @usual_traitement_time = @procedure.stats_usual_traitement_time
+      @dossiers_funnel = @procedure.stats_dossiers_funnel
+      @termines_states = @procedure.stats_termines_states
+      @termines_by_week = @procedure.stats_termines_by_week
+
+      render 'statistiques/show'
+    end
+
+    private
+
+    def retrieve_procedure
```

Comment:
```suggestion
    def procedure
```

D'habitude on donne le nom procedure directement